pandas/core/frame.py

```diff
@@ -5651,12 +5651,32 @@ def isin(self, values):
                                 values).reshape(self.shape), self.index,
                 self.columns)
 
+    # ----------------------------------------------------------------------
+    # Add plotting methods to DataFrame
+    plot = base.AccessorProperty(gfx.FramePlotMethods, gfx.FramePlotMethods)
+    hist = gfx.hist_frame
+
+    @Appender(_shared_docs['boxplot'] % _shared_doc_kwargs)
```

Comment:
can you define boxplot *in* ``pandas.plotting._core`` (its already imported as ``gfx``)
since ``boxplot`` is already defined there (you are calling it in fact), would need to name this differently
(boxplot_frame)